modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.2.0.adoc

```diff
@@ -35,7 +149,9 @@ None for this release
 
 ===  Known Issues
 
-None for this release
+* https://issues.couchbase.com/browse/CBL-5551[CBL-5551 - Index trained regardless of warning when using PQ with hihg subquantizer]
```

Comment:
This issue has been already fixed in beta-3.

Reply:
Awesome thank you!